src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { Store, MutationPayload } from "vuex";
 import merge from "deepmerge";
 import * as shvl from "shvl";
+import _ from "lodash";
```

Comment:
you'll need to npm intall lodash in order to use this

Reply:
oops